geopandas/_compat.py

```diff
@@ -22,6 +22,16 @@
 PANDAS_GE_14 = Version(pd.__version__) >= Version("1.4.0rc0")
 
 
+# -----------------------------------------------------------------------------
+# fiona compat
+# -----------------------------------------------------------------------------
+try:
+    import fiona
+
+    FIONA_GE_19 = Version(Version(fiona.__version__).base_version) >= Version("1.9.0")
```

Comment:
I think we should not do this in compat.py, as that means we will always try to import fiona on geopandas import, while we tried hard to not do that (https://github.com/geopandas/geopandas/pull/2427). Can you move this to inside the `_import_fiona` function?